types/src/transaction/write_set.rs

```diff
@@ -0,0 +1,25 @@
+// Copyright (c) The Libra Core Contributors
+// SPDX-License-Identifier: Apache-2.0
+
+use crate::{contract_event::ContractEvent, write_set::WriteSet};
+use serde::{Deserialize, Serialize};
+
+#[derive(Clone, Debug, Hash, Eq, PartialEq, Serialize, Deserialize)]
+pub struct WriteSetPayload {
```

Comment:
Any suggestions for a better name here? It seems to me that calling it a `ChangeSet` makes more sense but I would like to hear your thoughts.

Reply:
I think that's fine

Reply:
`TransactionEffect` is another idea.